src/components/LoanCards/LoanProgressGroup.vue

```diff
@@ -33,15 +33,32 @@ export default {
 			type: String,
 			default: '',
 		},
+		enableLoanCardExp: {
+			type: Boolean,
+			default: false
+		}
 	},
 	computed: {
+		numeralLeft() {
+			return numeral(this.moneyLeft);
+		},
+		amountLow() {
+			// Changing UI state based on amount left is currently under experiment
+			return this.enableLoanCardExp && this.numeralLeft.value() < 100;
+		},
 		fundingText() {
-			const formattedMoneyLeft = numeral(this.moneyLeft).format('$0,0[.]00');
+			const formattedMoneyLeft = this.numeralLeft.format('$0,0[.]00');
 			const formattedTimeLeft = `${this.timeLeft !== '' ? `. ${this.timeLeft}` : ''}`;
-
-			const formatttedFundingText = `${formattedMoneyLeft} to go${formattedTimeLeft}`;
-			return 	formatttedFundingText;
+			// Some time left strings already include an exclamation mark
+			const exclamationMark = this.amountLow && !formattedTimeLeft.includes('!') ? '!' : '';
+			return `${formattedMoneyLeft} to go${formattedTimeLeft}${exclamationMark}`;
 		}
 	}
 };
 </script>
+
+<style scoped>
+.progress-group-amount-low {
+	color: #FF5C00;
```

Comment:
What's the acccessibilty score for this color over a white bg?

Reply:
Interestingly, I just ran Lighthouse locally, and it called out colors in the footer, not this new orange. Is there another test you'd recommend?

Reply:
Thanks for checking that! I love orange and white together but it buzzes on the page a little oddly with text...the footer stuff was known.

Reply:
Agreed about the lightness of the color buzzing. I guess we'll see how the experiment performs.